VARIABLES.md

```diff
@@ -8,7 +8,7 @@ Below are the supported variables for the role variables
 
 Version of Confluent Platform to install
 
-Default:  6.2.0
+Default:  6.2.1
```

Comment:
This  variable is controlled by automation. Please rebase to latest changes in this branch.

Reply:
you want me to rebase to 6.2.0 or 7.0.x?